ExampleService/ExampleService_WebApi/Controllers/TaskController.cs

```diff
@@ -0,0 +1,53 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace ExampleService_WebApi;
+
+[ApiController]
+[Route("[controller]")]
+public class TaskController :  ControllerBase
+{   
+    private readonly InterfaceClass _interface;
+
+    //injecting the interface dependency
```

Comment:
It is not needed to add the comment if the code itself is self explanatory. Typically we should write the code in a way the it doesn't need any comments to understand it. 

Where we should add comment,
1. For APIs and models, since it will be exposed in swagger and anyone uses the swagger should know what that API does.
2. For Interface and its methods.
3. For complex logic where it is not easy to understand just by looking at the code

Reply:
Removed unnecessary documentation.